client/api.go

```diff
@@ -25,7 +25,12 @@ func Host() string {
 
 func HostURL() (*url.URL, error) {
 	apiURL := os.Getenv("FN_API_URL")
+
 	if apiURL == "" {
+		if os.Getenv("API_URL") != "" {
+			fmt.Fprint(os.Stderr, "Error: API_URL is deprecated, please use FN_API_URL.")
```

Comment:
this might be just a warning, don't have to be so critical

Reply:
I mean that suppose to be enough to write the message to STDERR like you did, I'm not sure that we should exit in case if FN_API_URL is not set.

Reply:
since it initializes on every command, it prints the warning like 20 times. I can fix by introducing a global bool but feels unnecessary .... this is all temporary until all current users switch to FN_API_URL and then we can just delete these lines.